backend/src/main/java/org/example/backend/service/GuestsService.java

```diff
@@ -16,37 +18,47 @@ public class GuestsService {
     private final IdService idService;
 
 
-    public List<Guests> getAllGuests() {
+    public List<GuestsModel> getAllGuests() {
         return guestsRepo.findAll();
     }
 
 
-    public Optional<Guests> getGuestById(String id) {
+    public Optional<GuestsModel> getGuestById(String id) {
         return guestsRepo.findById(id);
     }
 
     public void addGuest(GuestDto guestDto) {
-        Guests guests = new Guests(idService.generateUUID(),
+        GuestsModel guests = new GuestsModel(idService.generateUUID(),
                 guestDto.name(),
                 guestDto.email(),
                 guestDto.rsvpStatus(),
-                guestDto.notes());
+                guestDto.notes(),
+                List.of());
         guestsRepo.save(guests);
     }
 
     public void updateGuest(String id, GuestDto guestDto) {
-        Guests updateGuest = guestsRepo.findById(id).orElseThrow();
+        GuestsModel updateGuest = guestsRepo.findById(id).orElseThrow();
         updateGuest = updateGuest.withName(guestDto.name())
```

Comment:
Ich sehe in deinem code an einigen Stellen das orElseThrow() ohne message. Wenn du hier eine message mitgeben würdest, fällt das spätere debugging leichter.
Auch fällt mir auf, dass du keine exceptions im backend handelst. Das würde ich dir dringend empfehlen, da du dann einfach schneller weißt wo es hängt, wenn etwas hängt.